lib/commands/debug.ts

```diff
@@ -1,18 +1,26 @@
 ﻿///<reference path="../.d.ts"/>
 "use strict";
 
-import helpers = require("./../common/helpers");
-import util = require("util")
+export class DebugPlatformCommand implements ICommand {
```

Comment:
Of course, it is exposed by the ios and android debug commands.
It would throw TS errors if it is not exported. 

I can copy the implementation of the base class into the two derived classes if you insist.